Configuration.cs

```diff
@@ -18,7 +18,7 @@ public class Configuration
         public int GlossAdjustmentIndex;
         public bool RaceMode;
         
-        private const string fileLocation = "Config.xml";
+        private string fileLocation = Application.StartupPath + @"\Config.xml";
```

Comment:
This was the main cause of the failed start via other context such as Windows search.
Without explicitly clarifying StartupPath it instead will reference the Command Path:

"C:\System32\windows\Config.xml" if spawned via Search